examples/correct/functions_basic.cac

```diff
@@ -2,14 +2,13 @@ let simple_check = [x: Int] -> Bool => x == 42; # the body of a function can be
 simple_check[43]; # functions can be called - returns `false`
 
 let complex_check = [x: Int] -> Bool => (
-    let temporary = simple[x];
+    let temporary = simple_check[x];
     temporary || x == 43
 ); # the body of a function can also be a block
 complex_check[43]; # returns `true`
 
-let global = false;
 let confusing = [x: Int] -> Bool =>
-    global = true; # careful! this is syntactically correct but the indentation is confusing
```

Comment:
Why did you remove global variable? Maybe it is not compiling right now (because we don't have static links to global scope, but it should compile in the future).

Reply:
I will move this part of code to "not yet correct" folder, since we won't be supporting global variables anytime soon